src/polkaswap-connect-with-google.md

```diff
@@ -0,0 +1,99 @@
+  - - meta
+    - name: keywords
+      content: "Polkaswap Google login, Polkaswap account, Google account, Polkaswap tutorial, connect account, Google login tutorial, SORA account, import account, JSON file, transaction security"
+---
+
+# Connect With Google
+
+Connecting to [Polkaswap.io ](https://polkaswap.io/#/wallet) has never been easier. Use a Google login to access your SORA account with style and freedom.
+This feature is for people who are new at DeFi and blockchain accounts.
+**Never share your personal account information with anyone**.
+
+![](.gitbook/assets/google-login-1.png)
+
+1. Navigate to the Account section in [Polkaswap.io ](https://polkaswap.io/#/wallet) and click the “Switch Account” button.
+
+![](.gitbook/assets/google-login-2.png)
+
+2. Click on Google and sign in with the existing account on your computer or create a new account.
+
+![](.gitbook/assets/google-login-3.png)
+
+3. Allow [Polkaswap.io ](http://Polkaswap.io) access to your Google account, after allowing this permission, Polkaswap will use the keys to your account, safely stored within your Google Drive.
+
+![](.gitbook/assets/google-login-4.png)
+
+4. Once you have successfully signed in, you can create a new account or import an account using a JSON file.
+
+![](.gitbook/assets/google-login-5.png)
+
+5. If you want to create a new account, click the button and store the 12-word phrase to secure your account, also known as seed or [mnemonic phrase](create-an-address.md#restoring-account-from-mnemonic-in-different-applications). Store it safely, and never share it with anyone!
+
+![](.gitbook/assets/google-login-6.png)
+
+6. Confirm your passphrase to secure your account, or skip the process. (You can return to confirm your passphrase later.
+
+![](.gitbook/assets/google-login-7.png)
+
+7. Give your account a name and password, then confirm the password. You can also export a JSON file at this point. **If you toggle the JSON file for the account restoration option, Polkaswap will download your account’s JSON file to your computer or device.**
+
+::: info
+You can use the JSON file to import an account on Polkaswap or another wallet.
+:::
+
+![](.gitbook/assets/google-login-8.png)
+
+Success! You have created a SORA account to use on Polkaswap using Google!
+
+![](.gitbook/assets/google-importing-4.png)
+
+To access Polkaswap using this account, click on the account’s name, then input the account password you set in the last step.
+
+**_You are now connected through Google with style and freedom!_**
+
+## Importing an Existing Google Account
+
+1. With the JSON file of the account you want to import easy to reach, navigate to the Account section on Polkaswap and click the Switch account button.
```

Comment:
`to import easy to reach` -- what do you mean? please rephrase

Reply:
```suggestion
1. With the JSON file of the account you want to import at hand, navigate to the Account section on Polkaswap and click the Switch account button.
```

Reply:
@outoftardis Is this better phrasing? I am trying to convey that the reader should have the JSON file ready to drag and drop

Reply:
yes, this is better